wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/settings/HelpScreen.kt

```diff
@@ -0,0 +1,95 @@
+package au.com.shiftyjelly.pocketcasts.wear.ui.settings
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.wear.compose.foundation.lazy.ScalingLazyListScope
+import androidx.wear.compose.material.MaterialTheme
+import androidx.wear.compose.material.Text
+import au.com.shiftyjelly.pocketcasts.wear.ui.component.ScreenHeaderChip
+import au.com.shiftyjelly.pocketcasts.wear.ui.component.WatchListChip
+import com.google.android.horologist.compose.layout.ScalingLazyColumn
+import com.google.android.horologist.compose.layout.ScalingLazyColumnState
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+object HelpScreen {
+    const val route = "help_screen"
+}
+
+@Composable
+fun HelpScreen(columnState: ScalingLazyColumnState) {
+
+    val viewModel = hiltViewModel<HelpScreenViewModel>()
+    val state = viewModel.state.collectAsState().value
+    val context = LocalContext.current
+
+    ScalingLazyColumn(columnState = columnState) {
+        item {
+            ScreenHeaderChip(text = LR.string.settings_title_help)
+        }
+
+        if (state == null) {
+            return@ScalingLazyColumn
+        } else if (state.isPhoneAvailable) {
+            phoneAvailableContent(
+                onEmailLogsToSupport = { viewModel.emailLogsToSupport(context) },
+                onSendLogsToPhone = { viewModel.sendLogsToPhone(context) },
+            )
+        } else {
+            noPhoneAvailableContent()
+        }
+    }
+}
+
+private fun ScalingLazyListScope.phoneAvailableContent(
+    onEmailLogsToSupport: () -> Unit,
+    onSendLogsToPhone: () -> Unit,
+) {
+    item {
+        Text(
+            text = stringResource(id = LR.string.settings_help_contact_support_wear_requires_nearby_phone),
+            textAlign = TextAlign.Center,
+            style = MaterialTheme.typography.caption3,
+            color = MaterialTheme.colors.onSecondary,
+        )
+    }
+
+    item {
+        Spacer(Modifier.height(8.dp))
+    }
+
+    item {
+        WatchListChip(
+            title = stringResource(LR.string.settings_help_contact_support),
+            onClick = onEmailLogsToSupport,
+        )
+    }
+
+    item {
+        WatchListChip(
+            title = stringResource(LR.string.settings_help_send_logs_to_phone),
+            onClick = onSendLogsToPhone,
+        )
```

Comment:
I don't reckon we need both options of "Contact Support" and "Send Logs To Phone", it seems a bit confusing to have both. But I will leave that decision with you.

Reply:
I went back and forth on this, but you're probably right that there's no point in having both, so I removed the send logs to phone option in [3658c43](https://github.com/Automattic/pocket-casts-android/pull/1088/commits/3658c4333d04d7bc05418f16ca05241cd87b01db).